tests/test_batch_crossover_integration.py

```
@@ -0,0 +1,391 @@
+"""
+Integration tests for batch crossover in genetic algorithm.
+
+This module tests that the batch crossover operator is properly integrated
+into the genetic algorithm's parallel processing pipeline.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)